dace/codegen/targets/rtl.py

```diff
@@ -0,0 +1,519 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
+
+import itertools
+
+from typing import List, Tuple, Dict
+
+import dace
```

Comment:
Please try to avoid importing `dace` if possible (this gives issues with cyclic dependencies): instead, import the specific submodules that you need (like you did on the following line)